src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -235,14 +252,25 @@ def set_ignored_validations(
     def get_ignored_validations(self, **kwargs) -> models.IgnoredValidations:
         return self.api_client.client.v2_get_ignored_validations(self.id, **kwargs)
 
-    def set_ocs(self, properties=None):
-        self.set_olm_operator(consts.OperatorType.OCS, properties=properties)
+    def set_ocs(self, properties: str = None, update: bool = False):
+        self.set_olm_operator(consts.OperatorType.OCS, properties=properties, update=update)
+
+    def set_cnv(self, properties: str = None, update: bool = False):
+        self.set_olm_operator(consts.OperatorType.CNV, properties=properties, update=update)
 
-    def set_cnv(self, properties=None):
-        self.set_olm_operator(consts.OperatorType.CNV, properties=properties)
+    def set_lvm(self, properties: str = None, update: bool = False):
+        self.set_olm_operator(consts.OperatorType.LVM, properties=properties, update=update)
 
-    def set_lvm(self, properties=None):
-        self.set_olm_operator(consts.OperatorType.LVM, properties=properties)
+    def set_mce(self, properties: str = None, update: bool = False):
+        self.set_olm_operator(consts.OperatorType.MCE, properties=properties, update=update)
+
+    def set_metallb(self, properties: str = None, update: bool = False):
```

Comment:
```suggestion
    def set_metallb(self, properties: Optional[str] = None, update: bool = False) -> None:
```
- If the string can be `None` then the type should be `Optional`
- Let's use typing also for the return type